File: Semester1/B-PSU-100/sokoban/display.c
/*
** EPITECH PROJECT, 2022
** B-PSU-100-STG-1-1-sokoban-florent.guittre
** File description:
** display
*/

#include <ncurses.h>
#include "./include/my_structs.h"

void display_map(map_t *map, int x, int y)
{
    if (map->height < y && map->width < x)
        printw(map->buffer);
    if (map->height >= y || map->width >= x)
        mvprintw(y / 2, ((x - 48) * (((x - 48) < 0) ? (-1) : 1)) / 2,
        "Your terminal is too small to display the map...");
}

File: Semester1/B-MAT-100/105torus/display.py
##
## EPITECH PROJECT, 2023
## B-MAT-100-STG-1-1-105torus-florent.guittre
## File description:
## display
##

def display(x, precision):
    if (len(str(x)) >= precision + 2):
        print(f"x = {x:.{precision}f}")
    else:
        print(f"x = {x}")
